commands/run.sh

```diff
@@ -40,8 +40,16 @@ prebuilt_service_overrides=()
 prebuilt_services=()
 
 # We look for a prebuilt images for all the pull services and the run_service.
+prebuilt_image_override="$(plugin_read_config RUN_IMAGE)"
 for service_name in "${prebuilt_candidates[@]}" ; do
-  if prebuilt_image=$(get_prebuilt_image "$service_name") ; then
+  prebuilt_image=$(get_prebuilt_image "$service_name") || :
+
+  # override run_service prebuilt_image if set
+  if [[ -n "$prebuilt_image_override" ]] && [[ "$service_name" == "$run_service" ]] ; then
+    prebuilt_image="$prebuilt_image_override"
+  fi
+
+  if [[ -n "$prebuilt_image" ]] ; then
     echo "~~~ :docker: Found a pre-built image for $service_name"
```

Comment:
this message should be updated because the override image code would make it look in logs as if a pre-built image was found

Reply:
Ah! good call